stanford_events.module

```diff
@@ -131,6 +131,22 @@ function stanford_events_views_post_render(ViewExecutable $view, &$output, Cache
 
 }
 
+/**
+ * Implements hook_views_pre_execute().
+ */
+function stanford_events_views_pre_execute(ViewExecutable $view) {
```

Comment:
This seems pretty heavy handed and will introduce more obscurity than it should. Perhaps we should investigate an alternative solution like rebuilding the view to satisfy the end result.

Reply:
I'm certainly open to that; I tried rebuilding the view in several ways, but when you build a view that lists paragraphs through entity_reference_revisions, it uses the `paragraphs_item_field_data` table as the base table for the view, and that table will include revisions which are no longer referenced on the Event node.

If you've got a more elegant approach, I'd be happy to implement it, but this is the least intrusive way I could find. 🤷

Reply:
https://github.com/SU-SWS/stanford_profile/pull/354

Reply:
Styles and some small tweaks probably still need to be done though.